src/test/java/be/seeseemelk/mockbukkit/WorldMockTest.java

```diff
@@ -869,6 +870,17 @@ void spawnFriendly_Peaceful()
 	}
 
 	@Test
+	void testSpawnAllay()
+	{
+		WorldMock world = new WorldMock(Material.DIRT, 3);
+		Entity entity = world.spawnEntity(new Location(world, 0, 0, 0), EntityType.ALLAY);
+		assertInstanceOf(AllayMock.class, entity);
+		assertTrue(entity.isValid());
+		assertFalse(entity.isDead());
+
+	}
```

Comment:
```suggestion
	}
```
`#isValid` simply returns the reverse of `#isDead`. No need to test both.